src/main/java/com/github/onsdigital/babbage/util/http/PooledHttpClient.java

```diff
@@ -149,20 +149,19 @@ private URI buildGetUri(String path, List<NameValuePair> queryParameters) {
     /**
      * Throws appropriate errors if response is not successful
      */
-    private CloseableHttpResponse validate(CloseableHttpResponse response, HTTP requestDetails) throws ClientProtocolException {
+    private CloseableHttpResponse validateResponse(HttpRequestBase request, CloseableHttpResponse response)
+            throws ClientProtocolException {
+        info().endHTTP(request, response).log("PooledHttpClient request completed");
```

Comment:
Just have a little discomfort from having "completed" as the first line in a method. This method therefore appears to be inferring knowledge of where it is being used.